packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockCaptureConnection.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A mock implementation of `FLTCaptureConnection` that allows injecting a custom implementation.
+class MockCaptureConnection: NSObject, FLTCaptureConnection {
+  var setVideoOrientationStub: ((AVCaptureVideoOrientation) -> Void)?
+
+  var connection: AVCaptureConnection {
+    preconditionFailure("Attempted to access unimplemented property: connection")
+  }
+  var isVideoMirrored = false
+  var _videoOrientation = AVCaptureVideoOrientation.portrait
+  var videoOrientation: AVCaptureVideoOrientation {
+    get {
+      return _videoOrientation
+    }
+    set {
+      if let stub = setVideoOrientationStub {
+        stub(newValue)
```

Comment:
why does it use both stub and a backing value?

Reply:
It was like this in objc mock, but it seems this field is never read in tests so I will remove the backing value